bucket/hourglass.json

```
@@ -0,0 +1,27 @@
+{
+    "homepage": "https://chris.dziemborowicz.com/apps/hourglass/",
+    "description": "The simple countdown timer for Windows",
+    "version": "1.10.0",
+    "license": "MIT",
+    "url": "https://github.com/dziemborowicz/hourglass/releases/download/v1.10.0/HourglassPortable.exe#/Hourglass.exe",
+    "hash": "d61981ef85b0b9b6142d30f6bc16f8bfd22efd477914ecd6a6794a5ecfd961b6",
+    "bin": "Hourglass.exe",
```

- Github code width is 139 chars, PR width is 127 chars. Full length of inline pre_install is 126 chars, which make it safe to render and review in one line
- Always sort properties in order of execution

```suggestion
    "pre_install": "if (!(Test-Path \"$persist_dir\\Hourglass.config\")) { New-Item \"$dir\\Hourglass.config\" | Out-Null }",
    "bin": "Hourglass.exe",
```